lib/app/shared/constants/constants_json.dart

```diff
@@ -324,7 +324,7 @@ abstract class ConstantsJson {
           'JsonWebSignature2020',
           'Ed25519Signature2018',
           'EcdsaSecp256k1Signature2019',
-          'RsaSignature2018',
```

Comment:
you should have a warning for this in vscode.

Reply:
yes. I got the warning.